dev/tests/hu.elte.txtuml.export.cpp.tests/src/hu/elte/txtuml/export/cpp/wizardz/CompileTests.java

```diff
@@ -183,7 +184,7 @@ public void exportTest() {
 		}
 	}
 
-	@Test(timeout=600000)
+	@Test(timeout=720 * 1000)
```

Comment:
Is there some reasoning behind this limit?

Reply:
I have introduced the limit in order to avoid the test deadlock cased by cpp runtime dedlock.  I have added the pingpong model to tests so the timout was incarsed as well. It is only an upper estimate to the running time of cpp tests.